### PR TITLE
PR to add comments to clarify how one should interpret the GCTA output

### PR DESCRIPTION
**Pull request by mmeier93**
## This PR does the following

It adds comments in both `main.nf` and in `usage.md` regarding how one should interpret the output of GTCA, in terms of expected number of cases and controls.

## Details

- Added a small comment in line 36 and 37 of `main.nf` to point to `docs/usage.md`
- Added details in `docs/usage.md` in section about `--gwas_cases` and `--gwas_controls` to explain how `GCTA` samples individuals and added a link to the GCTA documentation which has a larger paragraph explaining how it works.

## Comments:

Wasn't sure whether to run a job but I did anyway: https://cloudos.lifebit.ai/app/jobs/5fbfbaa1cc96a201129afa90

### CODE REVIEW

File: docs/usage.md [hunk 1]
```diff
@@ -39,11 +39,11 @@ nextflow run main.nf --num_participants 40
 
 - **--simulate_plink**: simulate PLINK files (default: false)
 
-- **--simulate_gwas_sum_stats**: simulate GWAS summary statistics with GCTA (default: false)
+- **--simulate_gwas_sum_stats**: simulate GWAS summary statistics with GCTA (default: false).
 
-- **--gwas_cases**: the number of cases to simulate for the GWAS summary statistics (the total with controls should match --effective_population_size)
+- **--gwas_cases**: the number of cases to simulate for the GWAS summary statistics (the total with controls should match --effective_population_size). In addition, it should be noted that the final number of simulated cases may be smaller than the number provided by the user, due to how `GCTA` samples individuals for the simulation (i.e. it selects a subset of individuals in the provided genotype data). See more details at: https://cnsgenomics.com/software/gcta/#GWASSimulation.
```
Comment (cgpu) [inline docs/usage.md:44]: Nice thanks for adding extra info 👍 !

**Review by cgpu — APPROVED** [review]
More docs, more awesomeness! 🍉 🍐 🍑

File: docs/usage.md (hunk 1 above)
Comment (mmeier93) [reply docs/usage.md:44]: 👍

**Comment by mmeier93** [comment]
> More docs, more awesomeness! 🍉 🍐 🍑

👍 😄

**Pull request merged**